### PR TITLE
we all know they do

### PR DESCRIPTION
**Pull request by KnorpelSenf**
so we tell them to just not

### CODE REVIEW

File: site/docs/id/guide/deployment-types.md [hunk 1]
```diff
@@ -236,6 +236,8 @@ app.use(webhookCallback(bot, "oak"));
 :::
 ::::
 
+> Kamu tidak perlu memanggil `bot.start()` ketika menggunakan webhooks.
```
Comment (fadzikri) [inline site/docs/id/guide/deployment-types.md:239]:
```suggestion
> Kamu tidak perlu memanggil `bot.start()` ketika menggunakan webhook.
```

Do we still need `s` in bahasa?

**Review by fadzikri — CHANGES_REQUESTED** [review]
@CikiMomogi what's your opinion about this?

**Review by LWJerri — APPROVED** [review]

File: site/docs/id/guide/deployment-types.md (hunk 1 above)
Comment (CikiMomogi) [reply site/docs/id/guide/deployment-types.md:239]: I am not sure though. What do you think?

Comment (KnorpelSenf) [reply site/docs/id/guide/deployment-types.md:239]: @fadzikri?

Comment (CikiMomogi) [reply site/docs/id/guide/deployment-types.md:239]: I think both of them is okay. But for the shake of consistency, we should determine which one should be used. This one is safe to be merged.

**Comment by KnorpelSenf** [comment]
Another thing that is blocked by #822

**Pull request merged**